common/config/rush/browser-approved-packages.json

```diff
@@ -2,6 +2,18 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
   "packages": [
+    {
+      "name": "@azure/core-htp",
```

Comment:
What is this? It looks like a typo, but you have core-http below, so I'm not sure.

Reply:
Weird. This is autogenerated. Will manually remove

Reply:
reverted